language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/contexts/RenameContextImpl.java

```diff
@@ -45,8 +45,9 @@
         super(operation, fileUri, wsManager, cursorPos, serverContext);
         this.clientCapabilities = clientCapabilities;
         this.params = params;
-        this.honorsChangeAnnotations = clientCapabilities.getTextDocCapabilities()
-                .getRename().getHonorsChangeAnnotations();
+        this.honorsChangeAnnotations = clientCapabilities.getTextDocCapabilities().getRename() != null &&
+                clientCapabilities.getTextDocCapabilities()
```

Comment:
Shall we reformat this into a single line:
```suggestion
                clientCapabilities.getTextDocCapabilities().getRename().getHonorsChangeAnnotations();
```

Reply:
Done